### DISCUSSION TITLE
Pipewire-jack

### DISCUSSION THREAD

**Discussion by mattdean3** [FrancescoCeruti/linux-show-player#314] [Q&A]
Running 0.6.1 on Ubuntu Studio 23.10. when I choose Jack as the output, I can see all of my devices in the Edit Connections window, and connect them, but when I try to play the cue, I get the message "lisp.plugins.gst_backend.gst_utils.GstError: ../libs/gst/base/gstbaseparse.c(3699): gst_base_parse_loop (): /GstPipeline:pipeline2/GstURIDecodeBin:uridecodebin2/GstDecodeBin:decodebin4/GstFlacParse:flacparse4:streaming stopped, reason not-linked (-1)".

**Comment by FrancescoCeruti** [comment]
I'm not sure what the problem is here, but pipewire-jack is not compatible with GStreamer see this issue: #286

**Comment by mattdean3** [comment]
Thanks for the response.
On my machine, I have 0.5.3 installed from the Ubuntu repositories, and 0.6.3 built from source. 0.5.3 works with pipewire-jack, but 0.6.3 shows error messages.

Apr 11, 2024 3:58:13 PM Francesco Ceruti ***@***.***>:

**Reply by FrancescoCeruti** [reply]
Mmmh, probably you're using an older version of pipewire that is still compatible.
Have you tried different file formats?
I'll try to reproduce the problem

**Comment by mattdean3** [comment]
Pipewire is version 0.3.79-2.
Gstreamer is version 1.22.5-1
Flac and Wave files produce the same results.

Apr 19, 2024 1:33:21 PM Francesco Ceruti ***@***.***>:
